xbmc/video/guilib/VideoSelectActionProcessor.cpp

```diff
@@ -17,28 +17,33 @@
 #include "utils/guilib/GUIContentUtils.h"
 #include "video/VideoFileItemClassify.h"
 #include "video/guilib/VideoGUIUtils.h"
+#include "video/guilib/VideoPlayActionProcessor.h"
 
 namespace KODI::VIDEO::GUILIB
 {
 
-Action CVideoSelectActionProcessor::GetDefaultSelectAction()
-{
-  return static_cast<Action>(CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(
-      CSettings::SETTING_MYVIDEOS_SELECTACTION));
-}
-
 Action CVideoSelectActionProcessor::GetDefaultAction()
 {
-  return GetDefaultSelectAction();
+  auto settings{CServiceBroker::GetSettingsComponent()->GetSettings()};
```

Comment:
Maybe add a to-do so this can be removed in future versions. Otherwise it might be forgotten.